app/views/tutorials/show.html.erb

```diff
@@ -11,13 +11,16 @@
     <%- end -%>
   <% end %>
 
-  <div class="sidenav-body tutorial">
+  <div class="sidenav-body tutorial" id="tutorial">
     <div class="article-intro">
       Get an overview of the online moving process with pro tips from experienced customers and counselors.<br>
       Need more? Visit your <%= link_to 'local transportation office', offices_path %> or call 1 (800) 462-2176 for 24/7 support.
     </div>
 
-    <h3><span>Tutorial</span> <%= @tutorial.title %></h3>
+    <div class="tutorial-header">
+      <h3><span>Tutorial</span> <%= @tutorial.title %></h3>
+      <a href="#" class="show-all" id="show-all"><%= image_tag 'symbols/list.svg', alt: 'show all steps icon' %></a>
```

Comment:
Since this anchor doesn't point to anything, I'd suggest making this element a `<button type="button">` and using CSS to get it to look right. You get the benefits of an interactive element without the implied semantics of an anchor element.